### ISSUE TITLE
client list shows old events

### ISSUE THREAD

**Issue by runningman84** [sensu/uchiwa#293]
The clients page shows old events even if they belong to checks which have been ok for a while (the event count is already 0). If you reload the clients page these old events dissapear.

**Comment by palourde** [comment]
Hi @runningman84 

Could you let me know which version of Uchiwa you are running? Also, what's the value of the *refresh* attribute under the *uchiwa* object in your configuration file?

If you open your web browser console, do you have any errors?

Thanks

**Comment by runningman84** [comment]
![sensu_error](https://cloud.githubusercontent.com/assets/1699128/6918842/195dc968-d7b2-11e4-850e-05d81872f895.PNG)

We are using:

sensu 0.17.0
uchiwa 0.7.0

This is the current config:
```
{
  "Sensu": [
    {
      "Name": "Sensu",
      "Host": "127.0.0.1",
      "Port": 4567,
      "Ssl": false,
      "Insecure": false,
      "URL": "http://127.0.0.1:4567",
      "User": "*****",
      "Path": "",
      "Pass": "*****",
      "Timeout": 5
    }
  ],
  "Uchiwa": {
    "Host": "127.0.0.1",
    "Port": 3000,
    "Refresh": 5,
    "Pass": "*****",
    "User": "*****",
    "Db": {
      "Driver": "",
      "Scheme": "*****"
    },
    "Auth": ""
  }
}
```
There are no erros in the web browser console.

**Comment by palourde** [comment]
Hi @runningman84 

Sorry for the delay. This issue has been resolved (see https://github.com/sensu/uchiwa/issues/309) and the fix will be available in the 0.8.0 release. Thanks
